app/models/differential_expression_result.rb

```diff
@@ -87,11 +87,72 @@ def filename_for(label, comparison: nil)
     "#{basename}.tsv"
   end
 
-  # map of all observed result files, of label value => label-specific filenames
+  # map listing all result files and their constituent groups , by comparison type
   # this is important as it sidesteps the issue of study owners renaming clusters, as cluster_name is cached here
+  #
+  # @return [Hash<String => Array<String, String>, Array<String, String, String>]
   def result_files
-    files = one_vs_rest_comparisons.map { |label| filename_for(label) }
-    Hash[one_vs_rest_comparisons.zip(files)]
+    pairwise_files = []
+    is_author_de = false
+
+    # TODO (SCP-5096): Productionize this block, remove example data
```

Comment:
Just a note: while this is totally fine for the purposes of this work, I would advocate for instead having a setup block in the MANUAL TESTING section of this PR to populate the required data into the `DifferentialExpressionResult` objects that we should have in our local databases.  It pollutes the code less and removes the need to take this out in a successive PR.

Example:
```
study = Study.find_by(name: 'Human milk - differential expression')
result = study.differential_expression_results.find_by(annotation_name: 'General_Celltype')
result.pairwise_comparisons = {
 ... required data
}
result.save!
```

Reply:
Yeah, I think this is a kludge almost any way we slice it.  The current approach is noisy, but seems easier to reason about and easier to remove (delete a clear block of code).  The proposed approach is cleaner in code, but harder to reason about, and requires more effort to add and especially to remove and restore to its original local state.

Reply:
Fair points, especially regarding cleanup.

Reply:
On second thought, having synthetic data seems like the conventional approach for this sort of thing.  I'll plan to incorporate such study data (and remove this block if it's still here) in my forthcoming PR for adding content to the DE table for author-computed DE data.